src/main/webapp/WEB-INF/themes/mobile/ftl/common/footContent.ftl

```diff
@@ -26,6 +26,6 @@
 
 <#include "siteMenu/siteMenu.ftl" />
 <#include "bodyJs.ftl" />
-
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```

Comment:
Is this moved down to prevent the asset from blocking the loading of JS?

Reply:
@sgatchell correct.  I ran some audits & it appeared to be a big offender, so I moved it.